### DISCUSSION TITLE
conda-forge CDN not syncing

### DISCUSSION THREAD

**Discussion by jakirkham** [conda/infra#643] [General]
It looks like it has been 6hrs+ since the last CDN sync for conda-forge. Could someone please take a look at this?

<img width="1840" alt="Screen Shot 2022-10-28 at 1 18 31 AM" src="https://user-images.githubusercontent.com/3019665/198539646-2f0c6fa7-223f-4f3d-b058-1c2dddb51dcc.png">

cc @chrisburr (for vis)

**Comment by jakirkham** [comment]
cc @jezdez @chenghlee

**Comment by jakirkham** [comment]
Related: https://github.com/conda/infra/discussions/644

**Comment by barabo** [comment]
Our clone VM ran out of space on it's root partition.  I'm enlarging it now, freeing up some space, then tuning the monitoring & alerting so we're not surprised by this again.

**Comment by barabo** [comment]
It's back online!  Our alerting didn't work, so I'm looking into why.  This shouldn't happen again.

**Reply by jakirkham** [reply]
Thanks Carl! 🙏 Can see a green status! 🤩

As to alerting, is this run on the instance that ran out of space or is this run on another machine? Asking as I'm wondering, what is the risk of the machine getting unmanageable to the point no notification makes it out? Is there another machine that checks in periodically and can deliver notifications?

<img width="1840" alt="Screen Shot 2022-10-28 at 8 40 09 AM" src="https://user-images.githubusercontent.com/3019665/198677770-1f885ee6-81c8-46cd-a402-0998fc01a89f.png">
